api-summary.js

```diff
@@ -361,6 +366,10 @@ class ApiSummary extends AmfHelperMixin(LitElement) {
     `;
   }
 
+  _sanitizeHTML(HTML) {
+    return sanitizer.sanitize(HTML, { ADD_ATTR: ['target'] })
```

Comment:
I am pretty sure that the result sometimes can be an object. I didn't figure out when this happens. In `arc-marked` I check whether the result is string and if not I have to call `toString()` on the result.

Reply:
It seems to be a [chrome issue](https://github.com/cure53/DOMPurify/issues/361). Adding validation, thanks @jarrodek.